### DISCUSSION TITLE
DownloadHelper Devolución o reembolso

### DISCUSSION THREAD

**Discussion by Ander0213** [aclap-dev/video-downloadhelper#1486] [License & Purchase requests]
### What's your issue?

He comprado la versión premium pero al estar en inglés no me di cuenta que compré la de Firefox y yo uso Chrome… Me imaginé que la extensión iba con cualquier navegador al ser eso una extensión entonces quiero resolverlo o al menos que sea de Chrome y así lo puedo usar también en mi móvil. Espero respuesta la he comprado hace una hora más o menos 

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi, please send a message to licenses@downloadhelper.net, we will refund you.

**Comment by Ander0213** [comment]
Hola buenas, a ver necesito el programa y la verdad no tengo idea de cm va
la versión premium, informándome bien según explican en su página al
parecer es mejor la versión de Firefox por permitir descargar vídeos de
YouTube.

Ahí veo que me han enviado la devolución es un buen servicio y rápido
aunque no tenga su propia página de atención al cliente, si quiero otra vez
comprar la versión para firefox tengo que hacer los mismos pasos y no pasa
nada que sea el mismo correo electrónico? Un saludo

**Reply by jcfrog** [reply]
The email is not important, the only important info is license key. Please send us an email.

**Reply by Mattmjs** [reply]
@Ander0213  It doesn't work on mobile.
